### ISSUE TITLE
scales are swapped relative to bands

### ISSUE THREAD

**Issue by beckermr** [esheldon/desimage#3]
I think the scales are swapped relative to the bands. Here they are done in the order of the bands:

https://github.com/esheldon/desimage/blob/f9606c08fb09e9dfbb1156f39027c1ad70296bfa/desimage/imagemaker.py#L342

but then when calling the maker only the images are reordered 

https://github.com/esheldon/desimage/blob/f9606c08fb09e9dfbb1156f39027c1ad70296bfa/desimage/imagemaker.py#L234

**Comment by esheldon** [comment]
Yes, the scales are rgb order because they can be set from the `des-make-image-fromfiles` command and I wanted to think in rgb rather than bgr

**Comment by beckermr** [comment]
Ahhh great. What scales do you recommend for lsst?

**Comment by beckermr** [comment]
I am still confused. The scales are supposed to be in RGB order, but in the snippet above, the exposure times are used in BGR order to adjust them.

**Comment by esheldon** [comment]
That could be a bug.

As for LSST I haven't worked on it, but probably won't be too far from the ones we use in DES.
